molecule/dynamic/group_vars/all

```diff
@@ -1,14 +0,0 @@
----
-pulp_default_admin_password: password
-pulp_install_db: true
```

Comment:
you need these variables for dynamic, you should move them to `../../example-use/group_vars/all`

Reply:
Not anymore. The idea is that dynaic users use the same non-expanding set of variables.